openrtb_ext/bidders.go

```diff
@@ -555,6 +577,8 @@ var bidderNameLookup = func() map[string]BidderName {
 	return lookup
 }()
 
+type NameNormalizer func(name string) (BidderName, bool)
```

Comment:
Nitpick: I think too much context is lost with the dropping of "bidder". Consider naming "BidderNameNormalizer" or "BidderNameFormatter".